packages/cli-kit/src/private/node/api/headers.ts

```diff
@@ -56,7 +56,7 @@ export function buildHeaders(token?: string): {[key: string]: string} {
     ...(firstPartyDev() && {'X-Shopify-Cli-Employee': '1'}),
   }
   if (token) {
-    const authString = token.match(/^shp(at|ua|ca)/) ? token : `Bearer ${token}`
+    const authString = token.match(/^shp(at|ua|tka)/) ? token : `Bearer ${token}`
```

Comment:
Need to include theme access tokens otherwise we put `Bearer shptka_123` in the header which is incorrect